src/sass/components/_footer.scss

```diff
@@ -83,9 +83,15 @@ footer {
   }
 }
 
-@media (max-width: 768px) {
-  
+@include media-breakpoint-down(sm) {
+
+body{
```

Comment:
Is this `body` selector necessary here? Actually, we have all tags inside `body`, so it's optional or even redundant.